applications/StructuralMechanicsApplication/custom_advanced_constitutive/generic_anisotropic_3d_law.cpp

```diff
@@ -620,7 +620,8 @@ void GenericAnisotropic3DLaw::CalculateCauchyGreenStrain(
     // Compute total deformation gradient
     const BoundedMatrixType& F = rValues.GetDeformationGradientF();
 
-    BoundedMatrixType E_tensor = prod(trans(F), F);
+    BoundedMatrixType E_tensor = ZeroMatrix(F.size1());
+    E_tensor = prod(trans(F), F);
```

Comment:
this is the important part. I just realized I also deleted some whitespaces ...

Reply:
shouldn't this be `noalias`?

Reply:
yes probably, I'll fix it

Reply:
replacing 
`E_tensor = prod(trans(F), F);`
with 
`noalias(E_tensor) = prod(trans(F), F);`

causes the same error :question:

Reply:
no idea...
Probably needs more investigation after this is merged @AlejandroCornejo

Reply:
mm this is a minor correction :)

`BoundedMatrixType E_tensor;`
`noalias(E_tensor) = prod(trans(F), F);`

And should be fine